django3_admin_select2/templates/admin/base_site.html

```diff
@@ -1,12 +1,12 @@
 {% extends "admin/base.html" %}
 
-{% load staticfiles %}
+{% load static %}
```

Comment:
General question, did you check the `base_stie.html` for Django3, it might have changed. Lets make sure we are inserting the latest.

Reply:
Yep, it has not changed and you can verify via the test app when this PR is merged ✔